CHANGELOG.md

```diff
@@ -198,7 +200,7 @@
 - Fix documentation for `ionoscloud_server`, `ionoscloud_volume`, `ionoscloud_lan` resources and `ionoscloud_image` data sources
 
 ### Enhancements
-- Add configurable fields to `ionoscloud_share` resource. Fields that can be set on creation: `description`, `sec_auth_protection`, `licence_type`.
+- Add configurable fields to `ionoscloud_share` resource. Fields that can be set on creation: `description`, `sec_auth_protection`, `licence_type`. 
```

Comment:
@cristiGuranIonos do you use some kind of formatting tool for this file? I'm just curious where these changes are coming from.

Reply:
no idea how they got added back. Maybe on merge in some way. Will leave as is, it does not matter.